src/whylogs/core/statistics/constraints.py

```diff
@@ -114,6 +114,17 @@ def update(self, v) -> bool:
             if self._verbose:
                 logger.info(f"value constraint {self.name} failed on value {v}")
 
+    def merge(self, other) -> "ValueConstraint":
```

Comment:
I've never seen type annotations as literal strings before, but it looks like it's valid.